### PR TITLE
Mat 5191 refactor measure version

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-5191](https://jira.cms.gov/browse/MAT-5191)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/test/java/cms/gov/madie/measure/resources/MeasureControllerTest.java [hunk 1]
```diff
@@ -57,7 +57,7 @@ public void setUp() {
     measure.setActive(true);
     measure.setMeasureSetId("IDIDID");
     measure.setMeasureName("MSR01");
-    measure.setVersion("0.001");
+    measure.setVersion(new gov.cms.madie.models.library.Version(0, 0, 1));
```
Comment (adongare) [inline src/test/java/cms/gov/madie/measure/resources/MeasureControllerTest.java]: is there any naming conflict that forces to use a fully qualified name?

File: src/main/java/cms/gov/madie/measure/resources/MeasureController.java [hunk 2]
```diff
@@ -35,6 +35,7 @@
 import cms.gov.madie.measure.services.GroupService;
 import cms.gov.madie.measure.services.MeasureService;
 import gov.cms.madie.models.common.ActionType;
+import gov.cms.madie.models.library.Version;
```
Comment (adongare) [inline src/main/java/cms/gov/madie/measure/resources/MeasureController.java]: I did not realize this earlier while reviewing models PR. Do you think this should be in the common package and not library specific?

Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/resources/MeasureController.java]: Sure will do. Thanks Ashok!

Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/resources/MeasureController.java]: @adongare I did the change in madie-java-models: https://github.com/MeasureAuthoringTool/madie-java-models/pull/85 Thanks!

Comment (adongare) [reply src/main/java/cms/gov/madie/measure/resources/MeasureController.java]: thank you!

**Review by adongare — APPROVED** [review]

**Pull request merged**